resources/validations/ui/patches/overmind-react+28.0.0.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/node_modules/overmind-react/package.json b/node_modules/overmind-react/package.json
```

Comment:
Honestly, I'm not certain. It is included in this large PR: https://github.com/cerebral/overmind/pull/503 It may be worth creating a smaller PR with just that fix.